src/ralph/models/xapi/video/video.py

```diff
@@ -0,0 +1,22 @@
+"""Video xAPI event definitions"""
```

Comment:
I would recommend to move `src/ralph/models/xapi/video/video.py` to `src/ralph/models/xapi/video/events.py`